src/AccessibilityInsights.Extensions.AzureDevOps/ConnectionInfo.cs

```diff
@@ -21,9 +20,9 @@ public class ConnectionInfo : IConnectionInfo
         [JsonIgnore]
         public Uri ServerUri => ConcreteServerUri;
         [JsonIgnore]
-        public IProject Project => ConcreteProject;
+        public TeamProject Project => ConcreteProject;
```

Comment:
Now that there's no longer an interface, I suspect we could simplify this (lose ConcreteProject and ConcreteTeam, have just Project and Team, and remove the [JsonIgnore] tags). That would break saved connection configs from existing users, but since we'll probably break them already, this doesn't seem so terrible. Doesn't necessarily need to be part of this PR, but if we release the build with the Concrete names in the serialized objects, we're sort of stuck with them for the long term

Reply:
Agreed, removed concrete fields.